src/transform/mod.rs

```diff
@@ -41,7 +41,7 @@ static INV_SQRT2: i32 = 2896; // 2^12 / sqrt(2)
 
 pub const TX_TYPES: usize = 16;
 
-#[derive(Debug, Copy, Clone, PartialEq, PartialOrd)]
+#[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Eq, Ord)]
```

Comment:
This also should go in the other PR.

Reply:
Similarly, this is only required to use this in a BTreeMap.